### PR TITLE
moved base_url to config.json

### PR DESCRIPTION
**Pull request by mccbryan3**

### CODE REVIEW

File: Agent-Install-Examples/eks_lumos_install_python/README.md [hunk 1]
```diff
@@ -19,11 +19,14 @@ NOTE: The API `base_url` info can be found when creating the API credentials in
...
 
+NOTE: The`api_base_url` will be provided when creating the API keys.
+
 ```
 {
         "falcon_client_id": "8972331hiusgdaugd987987231213",
         "falcon_client_secret": "cdHJKASDsdad7t8sdsajhkasdasd",
-        "falcon_cid": "JHDSHJDHFS9879EHKSJDAHKJHFA-JH"
+        "falcon_cid": "JHDSHJDHFS9879EHKSJDAHKJHFA-JH",
+        "api_base_url": "https://api.us-2.crowdstrike.com"
```
Comment (redhatrises) [inline Agent-Install-Examples/eks_lumos_install_python/README.md:29]: Recommend that we keep this consistent and call this `cloud` or `cloud_region`. See https://github.com/CrowdStrike/falcon-integration-gateway/blob/94dba6023807deae786036bed3a025778d4881a0/fig/falcon/api.py#L19

**Review by redhatrises — CHANGES_REQUESTED** [review]

File: Agent-Install-Examples/eks_lumos_install_python/README.md (hunk 1 above)
Comment (mccbryan3) [reply Agent-Install-Examples/eks_lumos_install_python/README.md:29]: how about `base_url` (that was my original name)?.. I really don't like coding the urls into the script as you may have to add/remove/change the urls over time which is why I am using the config.json, which is already known in the falconpy sdk. 

The falconpy sdk uses the term variable `base_url`. Also when creating the keys the UI says `BASE URL` as a copyable item.

What say you @redhatrises?

Comment (jshcodes) [reply Agent-Install-Examples/eks_lumos_install_python/README.md:29]: We use base_url in a lot of places, aligns to falconpy as well.

File: Agent-Install-Examples/eks_lumos_install_python/README.md [hunk 2]
```diff
@@ -19,11 +19,14 @@ NOTE: The API `base_url` info can be found when creating the API credentials in
...
 
+NOTE: The`base_url` will be provided when creating the API keys.
+
 ```
 {
         "falcon_client_id": "8972331hiusgdaugd987987231213",
         "falcon_client_secret": "cdHJKASDsdad7t8sdsajhkasdasd",
-        "falcon_cid": "JHDSHJDHFS9879EHKSJDAHKJHFA-JH"
+        "falcon_cid": "JHDSHJDHFS9879EHKSJDAHKJHFA-JH",
+        "base_url": "https://api.us-2.crowdstrike.com"
```
Comment (shawndwells) [inline Agent-Install-Examples/eks_lumos_install_python/README.md:29]: Are these real client ID, secret, and CIDs?

Comment (shawndwells) [reply Agent-Install-Examples/eks_lumos_install_python/README.md:29]: Looked closer and noticed the `asdasd` bits 😌 

Believe there is a semi-standardized string for these we've been using in code and docs. @redhatrises do you have them?

Comment (redhatrises) [reply Agent-Install-Examples/eks_lumos_install_python/README.md:29]: For CID, the standardized string is: `1234567890ABCDEF1234567890ABCDEF-12`. Should come up with something for id and secret.

Comment (mccbryan3) [reply Agent-Install-Examples/eks_lumos_install_python/README.md:29]: I will fix it up

**Review by jshcodes — APPROVED** [review]

**Pull request merged**